src/axom/primal/geometry/Tetrahedron.hpp

```diff
@@ -171,18 +187,76 @@ class Tetrahedron
    * \brief Returns the signed volume of the tetrahedron
    * \sa volume()
    */
+  AXOM_HOST_DEVICE
   double signedVolume() const
   {
-    const double scale = 1. / 6.;
+    constexpr double scale = 1. / 6.;
```

Comment:
is this called alot in device code when it is used? since division is expensive, it may be better to collect a set of numerical constants in core; etc. one_sixth = 0.16666666666666666666666666.... and use that in places like this.

Reply:
Nevermind. I blanked on the fact you marked it constexpr. Sorry for the noise.

Reply:
I'm reasonably sure that the compiler will handle this for us with `constexpr`.
The following is not perfect, but should be a reasonable approximation:

https://godbolt.org/z/aoWMTKj7e

![image](https://user-images.githubusercontent.com/5626552/148604141-fa19c640-5b25-498b-8c38-a6377c57e69c.png)
e.g. with `-O0`, it gets all three expressions: `constexpr double`, `const double` and `double`

Reply:
Yeah, I get it. I was in webex when I entered my initial comment and wasn't thinking.